book/src/getting_started.md

```diff
@@ -4,17 +4,32 @@ These instructions will help you set up a working Amethyst game development
 environment. Then we'll test this environment by compiling a simple "Hello,
 World" application.
 
-Amethyst should compile hassle-free on any platform properly supported by the
-Rust compiler. Here are the system requirements (they're pretty modest):
-
-* Minimum:
-  * CPU: 1GHz, the more cores the better
-  * RAM: 512 MiB
-  * HDD: 17 MiB free disk space
-  * OS: Windows Vista and newer, Linux, BSD, Mac OS X
-  * Rust: Nightly (1.6.0 or newer)
-* Renderer Backends:
-  * OpenGL 4.0 or newer
+Amethyst officially supports the following Rust targets:
+
+### Windows
+x86_64-pc-windows-msvc
+
+i686-pc-windows-msvc
+
+x86_64-pc-windows-gnu
+
+i686-pc-windows-gnu
+
+### Mac OS
+x86_64-apple-darwin
+
+i686-apple-darwin
```

Comment:
Speaking of which, we should test OS X on Travis.

Reply:
Agreed. Let's prepare a separate PR for that soon.